From 0376a840c82297a517d80d7b29efb619d0e4bcb7 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:05 -0400
Subject: [redhat] kabi: Adding symbol down_interruptible (kernel/semaphore.c)

Message-id: <c6a260f8397f63a9af7e943b981683e254b7faf3.1410510773.git.poros@redhat.com>
Patchwork-id: 96120
O-Subject: [PATCH RHEL7.1 065/233] kabi: Adding symbol down_interruptible (kernel/semaphore.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Used to acquire a semaphore lock which is required as vxfen is a multi threaded re-entrant module

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/down_interruptible
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/down_interruptible
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/down_interruptible

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/down_interruptible b/redhat/kabi/kabi-module/kabi_ppc64/down_interruptible
new file mode 100644
index 0000000..88b83c4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/down_interruptible
@@ -0,0 +1,2 @@
+#1-
+0x4289f7df down_interruptible vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/down_interruptible b/redhat/kabi/kabi-module/kabi_s390x/down_interruptible
new file mode 100644
index 0000000..24181ec
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/down_interruptible
@@ -0,0 +1,2 @@
+#1-
+0xae2a3f38 down_interruptible vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/down_interruptible b/redhat/kabi/kabi-module/kabi_x86_64/down_interruptible
new file mode 100644
index 0000000..598771e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/down_interruptible
@@ -0,0 +1,2 @@
+#1-
+0x9f6b4c62 down_interruptible vmlinux EXPORT_SYMBOL
-- 
1.7.1